test/rxd/testutils.py

```diff
@@ -126,3 +128,18 @@ def compare_data(data):
     )
     max_err = numpy.amax(abs(corr_vals.T - tst_dat[t2_0:t2_n, 1:]))
     return max_err
+
+
+def check_platform():
+    """Check whether there is an issue with the test on the current platform"""
+    try:
+        test_platform = platform.freedesktop_os_release()
+
+        # skip Ubuntu 22 and above
+        return test_platform.get("ID") == "ubuntu" and Version(
+            test_platform["VERSION_ID"]
+        ) > Version("22")
+
+    # not a Linux variant (or a supported one anyway)
+    except Exception:
```

Comment:
This has a tendency of catching a lot. Including typos in variable names. Could this be made more specific or solved without catching?